### PR TITLE
restore dashboard url path to dashboard index widget

### PR DESCRIPTION
**Pull request by elisechant**

### CODE REVIEW

**Comment by elisechant** [comment]
@tobyhede wondering if there is actually Rails route helpers that could be used instead?

File: app/views/dashboards/index.html.erb [hunk 1]
```diff
@@ -17,7 +17,7 @@
               <p>Updated at <%= dashboard.published_at.strftime("%d %b %Y %H:%M:%S") %></p>
             </div>
           </div>
-          <a href="<% dashboard_path(dashboard) %>" class="card__link__visible">View <% dashboard.name %></a>
```
Comment (elisechant) [inline app/views/dashboards/index.html.erb]: ahh.. i'm assuming this was the route helper

File: app/views/dashboards/index.html.erb [hunk 2]
```diff
@@ -17,7 +17,7 @@
...
+          <a href="<%= dashboard.dashboard_path %>" class="card__link__visible">View <% dashboard.name %></a>
```
Comment (tobyhede) [inline app/views/dashboards/index.html.erb]: there's a helper. I think i fixed this in my show branch as well

**Comment by tobyhede** [comment]
So I think I fixed this already

**Comment by elisechant** [comment]
ok, closing because no longer relevant

**Pull request closed without merging**